src/components/diamond.jsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Angle = '30deg';
+const Width = 32;
+const Height = Width + 5;
+const ShiftPhoto = Height / 2;
+
+const EmptyDiamond = styled.div`
+    width: ${(props) => props.size || Width}rem; 
+    height: ${(props) => props.size + 5 || Height}rem; 
+`;
+
+const DiamondLeft = styled.div`
+    transform: skewY(-${(props) => props.angle || Angle});
+
+    width: ${(props) => props.size || Width}rem;
+    height: ${(props) => props.size + 5 || Height}rem;
```

Comment:
this is Rita's calculations for the size proportions of the diamond with 30deg angle